Add Shelly push update troubleshooting list (#28470)
home-assistant/home-assistant.github.io: source/_integrations/shelly.markdown

We recommend using `unicast` for communication. To enable this, enter the local IP address of the Home Assistant server and port `5683` into the **CoIoT peer** field and push **SAVE** button. **This is mandatory for battery operated devices**. After changing the **CoIoT peer**, the Shelly device needs to be manually restarted.

> Home Assistant from version 2023.8.0 will display a repair issue for the Shelly device if push updates from this device do not reach the Home Assistant server. It may seem that the Shelly device was working properly before, and yet Home Assistant will display a repair issue, it's just that the user has not been informed about the problems so far.

Reviewer: ```suggestion
Home Assistant will display a repair issue for the Shelly device if push updates from this device do not reach the Home Assistant server.
```

Reviewer: I think it would be better not to refer to a specific HA version, as far as I know docs always describe the  current HA version

The list below will help you diagnose and fix the problem:

- Check if your Shelly devices have a properly configured `CoIoT peer`.

> - If you can't find the `CoIoT peer` settings in the device's web panel, it's probably using a very old firmware version and you should update it to the current one.

Reviewer 2: ```suggestion
- If you can't find the `CoIoT peer` settings in the device's web panel, it's probably using an ancient firmware version, and you should update it to the current one.
```

- If Shelly devices are in a different subnet than the Home Assistant server, you should ensure communication on `UDP` port `5683` between these subnets.
- If Home Assistant is running as a virtual machine or service on an operating system other than Home Assistant OS, you should open `UDP` port `5683` on the device's firewall and/or ensure that communication from this port is redirected to the Home Assistant service.

> - The lack of push updates may be related to the range of the WiFi network. If you are using a WiFi network with several access points, enable **Internet & Security** >> **WiFi Client AP Roaming** option. Consider moving Shelly device closer to the WiFi access point if possible. Consider adding an additional WiFi access point, which will improve the quality of connection with the device.

Reviewer 2: ```suggestion
- The missing push updates may be related to the WiFi network range. If using a WiFi network with several access points, enable **Internet & Security** >> **WiFi Client AP Roaming** option. Consider moving Shelly device closer to the WiFi access point. Consider adding another WiFi access point, which will improve the connection quality with the device.
```

> - If you think your Shelly devices are working properly and you don't want to make changes to your network/configuration you can ignore the repair issue but you must be aware that you are giving up the best experience of using first generation Shelly devices with Home Assistant.

Reviewer 2: ```suggestion
- If you think your Shelly devices are working correctly and don't want to change your network/configuration, you can ignore the repair issue. Still, you must know you are giving up the best experience of using first-generation Shelly devices with Home Assistant.
```
